engine/class_modules/priest/sc_priest_shadow.cpp

```diff
@@ -1553,6 +1614,38 @@ struct vampiric_embrace_t final : public priest_buff_t<buff_t>
   }
 };
 
+struct dark_thoughts_t final : public priest_buff_t<buff_t>
+{
+  dark_thoughts_t( priest_t& p ) : base_t( p, "dark_thoughts", p.find_spell( 341205 ) )
+  {
+    this->set_max_stack( 5 );
+    this->set_duration( timespan_t::from_seconds( 6 ) );
```

Comment:
same as above, this should either be in spell data or duration should be pulled automagically

Reply:
Testing without manually setting these does not appear to be getting it from the spelldata, it appears to not currently be in there or I'm missing something.

Updated the p.find_spell to instead get by name however in the most recent commit.